app/api/migrations/0004_question_and_answer_creation.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.1.7 on 2019-07-09 18:36
```

Comment:
Consider squashing the migrations into one migration which lights up the Q&A feature.

Reply:
done